src/vos/vos_internal.h

```diff
@@ -284,6 +292,10 @@ struct vos_container {
 	uint64_t		vc_io_nospc_ts;
 	/* The (next) position for committed DTX entries reindex. */
 	umem_off_t		vc_cmt_dtx_reindex_pos;
+	/* The epoch for the latest committed solo DTX. Any solo
+	 * * transaction with older epoch must has been committed.
+	 */
+	daos_epoch_t		vc_solo_dtx_epoch;
```

Comment:
It looks to me this 'vc_solo_dtx_epoch' should be per pool. (in vos_pool but not vos_container)

Reply:
Per pool also work, but current DTX table is per container based, so the vc_solo_dtx_epoch keeps the same.